package.json

```diff
@@ -185,5 +185,8 @@
       "jest",
       "babel-jest"
     ]
+  },
+  "dependencies": {
+    "query-string": "^6.1.0"
```

Comment:
This should be a dep of `bpk-docs` right?

Reply:
Yes it should - good catch!